.github/workflows/release.yml

```
@@ -175,56 +293,111 @@
           VERSION_NUM=$(echo "$VERSION" | sed 's/^v//')
           
-          # Create comprehensive release notes
-          cat > release_notes.md << 'NOTES_EOF'
-          ## 🚀 What's New in TiLoKit $VERSION
+          echo "📝 Extracting release notes for version $VERSION_NUM"
+          
+          # Initialize counters for features and fixes
+          FEATURE_COUNT=0
+          BUGFIX_COUNT=0
+          IMPROVEMENT_COUNT=0
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)